js/data.js

```diff
@@ -2,7 +2,22 @@
 
 var data = {
   view: 'entry-form',
-  entries: [],
+  entries: [
+    {
+      title: 'Ada Lovelace',
+      image: './images/placeholder-image-square.jpg',
+      notes: `Augusta Ada King, Countess of Lovelace was an English mathmatician and writer, chiefly known for her
+      work on Charles Babbage's proposed mechanical general-purpose computer, the Analytical Engine.`,
+      nextEntryID: 1
+    },
+    {
+      title: 'JavaScript',
+      image: './images/placeholder-image-square.jpg',
+      notes: `JavaScript, often abbreviated as JS, is a programming language that conforms to the ECMAScript
+              specification.`,
+      nextEntryID: 2
+    }
+  ],
```

Comment:
Here you shouldn't have any hard coded entries, entries should start as an empty array.